tools/gen-device-svd/gen-device-svd.go

```diff
@@ -537,7 +537,14 @@ func parseBitfields(groupName, regName string, fieldEls []*SVDField, bitfieldPre
 				enumName = strings.ToUpper(enumName)
 			}
 			enumDescription := strings.Replace(enumEl.Description, "\n", " ", -1)
-			enumValue, err := strconv.ParseUint(enumEl.Value, 0, 32)
```

Comment:
I see, this doesn't work in Go before version 1.13 due to binary literals being new?

Reply:
Correct.